pkgs/applications/audio/sony-headphones-client/default.nix

```diff
@@ -22,6 +22,7 @@ stdenv.mkDerivation rec {
   postPatch = ''
     substituteInPlace Constants.h \
       --replace "UNKNOWN = -1" "// UNKNOWN removed since it doesn't fit in char"
+      --replace "OUT_OF_RANGE = -1" "// OUT_OF_RANGE removed since it doesn't fit in char"
```

Comment:
The previous line is missing a trailing backslash.

Reply:
you are completely right